meta-intel-edison-bsp/recipes-bsp/u-boot/u-boot-common_2018.01.inc

```diff
@@ -0,0 +1,23 @@
+HOMEPAGE = "http://www.denx.de/wiki/U-Boot/WebHome"
+SECTION = "bootloaders"
+
+LICENSE = "GPLv2+"
+LIC_FILES_CHKSUM = "file://Licenses/README;md5=a2c678cfd4a4d97135585cad908541c6"
+PE = "1"
+
+# We use the revision in order to avoid having to fetch it from the
+# repo during parse
+
+# andy-shev's repo provides both u-boot with and without acpi enabled 
+#tag acpi (acpi enabled)
+SRCREV = "f5d7d4a7027fae6667fd4224cdf24fdc87c8f5f0"
+# tag edison-v2018-01 (acpi disabled)
+#SRCREV = "409609841024bff9ab96e3480b03ba357bc2261b"
+SRC_URI = "git://github.com/andy-shev/u-boot.git;branch=acpi;protocol=https"
+
+SRC_URI += "file://${MACHINE}.env"
+SRC_URI += "file://target_env/"
+# path to disabled acpi table generation on otherwise acpi enabled u-boot (for testing)
+#SRC_URI += "file://0001-Revert-x86-edison-Switch-to-ACPI-mode.patch"
+PV = "u-boot-2018.01"
```

Comment:
Are you sure you want to have `u-boot-` prefix here? That's something I'd rather see as the package name (`$PN`) instead, otherwise given the default package naming rules, the full package name would be `u-boot-u-boot-2018.01`.

I'd suggest replacing this with `2018.01` or leaving it out altogether - the recipe name would provide `$PV` in that case.

Reply:
Here's the example of the package name I see right now during the build: `u-boot-fw-utils-1_u-boot-2018.01-r0`, that second `u-boot` is apparently surplus.

Reply:
You are right, will modify